src/aat/resources/features/F-050/S-298.td.json

```diff
@@ -0,0 +1,40 @@
+{
+  "title": "must not validate when field validation fails",
+  "_guid_": "S-298",
+  "_extends_": "F-050_Test_Data_Base",
+
+  "specs": [
+    "a detailed profile in CCD",
+    "uses a value that exceeds the field max limit",
+    "has the 422 return code"
+  ],
+  "request": {
+    "body" : {
+      "data": {
+        "PhoneUKField": "012341234123"
+      }
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_422_Response",
+    "headers": {
+      "Transfer-Encoding" : "chunked"
+    },
+    "body": {
+      "exception" : "uk.gov.hmcts.ccd.endpoint.exceptions.CaseValidationException",
+      "timestamp" : "[[ANY]]",
+      "message" : "Case data validation failed",
+      "path" : "[[ANY]]",
+      "details" : {
+        "field_errors" : [
+          {
+            "id": "PhoneUKField",
+            "message": "The data entered is not valid for this type of field, please delete and re-enter using only valid data"
+          }
+        ]
+      },
+      "callbackErrors" : null,
+      "callbackWarnings" : null
```

Comment:
please check and remove tall he common fields that are now coming from base object.

Reply:
Removed callback error & warning.